src/dataset-builder/DatasetBuilderHeader.ts

```diff
@@ -15,14 +14,15 @@ interface BuilderPageHeaderProps {
 
 const PAGE_PADDING_HEIGHT = 0;
 const PAGE_PADDING_WIDTH = 3;
+const DATASET_NAME = 'AXIN';
```

Comment:
Nit: `AxIN`, I believe? Also, do we want to hardcode the name? This seems like something we may want to fetch from the data structure itself.

Reply:
I'm not sure - I checked the SOW and they have it with all caps. But, that might not be right. I can check with Victoria. 

Yeah, I agree - just stubbing this data out until we figure out a way to return it from api - https://github.com/DataBiosphere/terra-ui/pull/4815#discussion_r1594613381

Reply:
The mocks have "AnalytixIN" so I'll do that for now.